src/main/java/com/flowingcode/vaadin/addons/localecombobox/LocaleComboBox.java

```diff
@@ -144,13 +181,22 @@ private Locale getLocaleForDisplay() {
   }
 
   private void onValueChange(ComponentValueChangeEvent<ComboBox<Locale>, Locale> event) {
+
+    if (!this.hasFlags)
```

Comment:
```
    if (this.hasFlags) {
       Locale newValue = event.getValue();
       this.setPrefixFlag(newValue);
    }
```

Reply:
i love my returns 😁

Reply:
First, note that braces are always used https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used, even when the body is empty or contains only a single statement.

I don't have an argument against functions with several exit points, but in this case I find the control flow questionable (it boils down to "unless has flags, do something" vs. "if has flags, do something")

Also, after applying formatting, the actual logic (when the method is called when there flags are enabled) takes less lines of code than the guard. (Such an `if...return;` may have simplified a more complex method where several structures would be nested, but this is not the case.

```diff
-   if (!this.hasFlags) {
-       return;
-   }   
-   Locale newValue = event.getValue();
-   this.setPrefixFlag(newValue)
+
+	if (this.hasFlags) {
+       Locale newValue = event.getValue();
+       this.setPrefixFlag(newValue)
+   }
```

Reply:
Well, we have no conventions on if/return `https://github.com/FlowingCode/DevelopmentConventions/issues/41` (but formatting must be applied in any case).

Reply:
I will add the curly braces, I wasn't aware that the style guide specified it